src/remix/sdk-setup.ts

```diff
@@ -36,6 +38,52 @@ export type PartialRemixConfig = {
 };
 
 const REMIX_CONFIG_FILE = 'remix.config.js';
+const REMIX_REVEAL_COMMAND = 'npx remix reveal';
+
+export function runRemixReveal(isTS: boolean): void {
+  // Check if entry files already exist
+  const clientEntryFilename = `entry.client.${isTS ? 'tsx' : 'jsx'}`;
+  const serverEntryFilename = `entry.server.${isTS ? 'tsx' : 'jsx'}`;
+
+  const clientEntryPath = path.join(process.cwd(), 'app', clientEntryFilename);
+  const serverEntryPath = path.join(process.cwd(), 'app', serverEntryFilename);
+
+  if (fs.existsSync(clientEntryPath) && fs.existsSync(serverEntryPath)) {
+    clack.log.info(
+      `Found entry files ${chalk.cyan(clientEntryFilename)} and ${chalk.cyan(
+        serverEntryFilename,
+      )}.`,
+    );
+  } else {
+    clack.log.info(
+      `Couldn't find entry files in your project. Trying to run ${chalk.cyan(
+        REMIX_REVEAL_COMMAND,
+      )}...`,
+    );
+
+    childProcess.execSync(REMIX_REVEAL_COMMAND, {
+      stdio: 'inherit',
```

Comment:
I don't know how `remix reveal` works but should we check for the return value or try/catch it? Just to ensure we don't crash but e.g. link to the manual setup if something goes wrong.

Reply:
This is handled on the `remix-wizard.ts`. This will throw on any non-zero exit code, which will be caught on the upper level.

Reply:
Ohh right, I missed this, thanks!